package.json

```diff
@@ -7,15 +7,20 @@
     "typecheck": "tsc"
   },
   "dependencies": {
+    "@unlock-protocol/contracts": "0.0.22",
+    "@unlock-protocol/networks": "0.0.18",
+    "@unlock-protocol/paywall": "0.6.6",
```

Comment:
A package that includes a UI for checkout... there again, you can replace with your own!